### PR TITLE
RLP implemented for basic types

### PR DESCRIPTION
**Pull request by akshatdev0**
RLP Encoding and Decoding functionality implemented for following types:
**`byte` `short` `int` `long` `byte[]` `BigInteger` `String`**
_Unit tests have been created for all._

### CODE REVIEW

**Comment by akshatdev0** [comment]
@Sarvesh19  @dineshal3287  @bhaskar6473 
Please provide any feedback regarding RLP.
**Also, DO NOT MERGE THE PULL REQUEST, WE WILL SQUASH THE COMMITS BEFORE MERGING**

File: src/main/java/studio/blockops/vyom/core/serialization/rlp/RLPEncoder.java
Comment (Sarvesh19) [inline src/main/java/studio/blockops/vyom/core/serialization/rlp/RLPEncoder.java]: I think we can put final here or?

File: src/main/java/studio/blockops/vyom/core/serialization/rlp/RLPEncoder.java
Comment (Sarvesh19) [inline src/main/java/studio/blockops/vyom/core/serialization/rlp/RLPEncoder.java]: here too

**Review by Sarvesh19 — APPROVED** [review]

**Pull request merged**
